Source/utils/display.cpp

```diff
@@ -278,16 +300,13 @@ void ReinitializeRenderer()
 	Size windowSize = { current.current_w, current.current_h };
 	AdjustToScreenGeometry(windowSize);
 #else
-	if (texture)
-		texture.reset();

```

Comment:
Better add these back. I got a failed assertion when changing the resolution in the settings menu. It may seem redundant to do it both here and in `ReinitializeTexture()`, but the texture does need to be destroyed before the renderer to avoid errors when we inevitably attempt to destroy the texture later on.

Reply:
I slipped this in with the integer scaling fixes.